src/adhocracy_mercator/adhocracy_mercator/scripts/manage_badges.py

```diff
@@ -0,0 +1,113 @@
+"""Assign badges to proposals.
+
+The functions of that script are registered as console script in
+setup.py.
+
+"""
+
+import inspect
+import argparse
+import transaction
+import json
+
+from pyramid.paster import bootstrap
+from pyramid.traversal import find_resource
+from adhocracy_core.resources.badge import IBadgeAssignment as BadgeResource
+from adhocracy_core.sheets.badge import IBadgeAssignment as BadgeSheet
+
+
+def add_badge_assignment():
+    """add badgeassignment.
+
+    usage::
+      bin/create_badge <config> <user> <badge> <proposal> <parent>
+    """
+    docstring = inspect.getdoc(add_badge_assignment)
+    parser = argparse.ArgumentParser(description=docstring)
+    parser.add_argument('ini_file',
+                        help='path to the adhocracy backend ini file')
+    parser.add_argument('user',
+                        type=str,
+                        help='path to user')
+    parser.add_argument('badge',
+                        type=str,
+                        help='path to badge')
+    parser.add_argument('proposal',
+                        type=str,
+                        help='path to proposal')
+    parser.add_argument('parent',
+                        type=str,
+                        help='path to parent')
+
+    args = parser.parse_args()
+    env = bootstrap(args.ini_file)
+
+    root = env['root']
+    registry = env['registry']
+
+    user = args.user
+    badge = args.badge
+    proposal_version = args.proposal
+    parent = args.parent
+
+    user = find_resource(root, user)
+    badge = find_resource(root, badge)
+    proposal_version = find_resource(root, proposal_version)
+    parent = find_resource(root, parent)
+
+    appstructs = {BadgeSheet.__identifier__:
+                  {'subject': user,
+                   'badge': badge,
+                   'object': proposal_version}}
+
+    registry.content.create(BadgeResource.__identifier__,
+                            parent=parent,
```

Comment:
BadgeAssignments have the be created in the post_pool 
defined in the badgeable sheet.
If don't want to use the sheet you can use `find_service(proposal, 'badge_assignments')`